python/pyspark/pandas/data_type_ops/base.py

```diff
@@ -206,3 +209,10 @@ def restore(self, col: pd.Series) -> pd.Series:
     def prepare(self, col: pd.Series) -> pd.Series:
         """Prepare column when from_pandas."""
         return col.replace({np.nan: None})
+
+    def isnull(self, element):
```

Comment:
Shall we add type annotations?

Reply:
Sure, it would be:
`def isnull(self, series) -> Union["Series", "Index"]:`